bin/s2nd.c

```diff
@@ -47,91 +47,92 @@
-    "c8lKvJWz6PRejf+xGn9JQU2HRgpsp5osnFuXDZCB/KkkVt1qcbDc/VEJ1tE7GLGr\n"
-    "mX+9beZQJj9dS43k5Qt9J9wrVifGGd/0sl+Or0hmoeV/qDJNsq8lHCjnrRZ+iSKt\n"
-    "DQ5NdV5z1lc5vOCARK0RyzBk07pHsfn39TCgyrIa+10cS6xrLSJcFC1eqC7sRRsW\n"
-    "cOVSt9nJDiKn+DOlEyMPWmwRUZX2rRzAlalDPsS/QzUUU3g+ExrBOACsySwiMKhX\n"
-    "aflRFDu7xHWKv9zK5FWysXjwTlKPOrDWcytNu+4NmWtBye/6s+5xyYjWbvdF\n"
-    "-----END CERTIFICATE-----\n"
-    "-----BEGIN CERTIFICATE-----\n"
-    "MIIDBDCCAeygAwIBAgIJAJB2OQ8SQr0KMA0GCSqGSIb3DQEBCwUAMBYxFDASBgNV\n"
-    "BAMMC3MyblRlc3RSb290MCAXDTE3MDIxMjA0MDk1M1oYDzIxMTcwMTE5MDQwOTUz\n"
-    "WjAWMRQwEgYDVQQDDAtzMm5UZXN0Um9vdDCCASIwDQYJKoZIhvcNAQEBBQADggEP\n"
-    "ADCCAQoCggEBANgSkebYTJIGo33QWSCKIwx8UXdOxXF9nljyQPQ6dgbuOTGbq4nF\n"
-    "eIA0AxMgkt0P8DnGuiJ/yuCvWmNAPy/wdmxPGxHfpDbiHFpbNGh10D7B3k68s51+\n"
-    "j4llBp3IymukQVFXQ4mFx3bI7VrSsGG9ZVd/+iMvSsU3yykhLRsgKgpj3RMYcb8C\n"
-    "WL3/lzR4F8QNZ9dwy0gifyF60xgC9n7D7kgI0PlDfIUBW68fjvscVhc6xYt1BDeG\n"
-    "++zngJpmeC9T7mN37/Nv5mXDCjKOtBGG9iOlriK6RzyPaUMc0n7gA1nB/mMy9GNe\n"
-    "lrMvIg44HSaKSdmwIbUVf96+aUegEexhrbsCAwEAAaNTMFEwHQYDVR0OBBYEFBSn\n"
-    "L6QRCOpg+LIFK3cubF0tZPOcMB8GA1UdIwQYMBaAFBSnL6QRCOpg+LIFK3cubF0t\n"
-    "ZPOcMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEBANEFXqBmaVoC\n"
-    "gOzPJol8saCkiIBJzgy4WpcY3TVM5z/MkpXd3fj0fR8bnhosmVmlHO6O2relGG+z\n"
-    "TtfU9fa7UG9fN+8fFjU59tw57qCJubKcoqJJzAIQCvCCuZ1yGGBkYwyCpKjQrkLX\n"
-    "54WNgAxXuPsQDzh4Y2943qk4kwliKEUPbkKnPuCDCxyHvpczXfbIdBcNKQ/WCkhW\n"
-    "ma7uqCaZ2sWx89UIR8ZVSpNuXAG+ZxvB1L5SN/8akCajK5z2AdVgAfTAKgsfb7WY\n"
-    "q76LCukvaANwJPnbsE0zHV5uYj6fl/V62msSaBlm1iI0glE6uZJAYZqeiEv3+cbF\n"
-    "e3r1R9CDhIo=\n"
-    "-----END CERTIFICATE-----\n";
+    "-----BEGIN CERTIFICATE-----"
+    "MIIDHTCCAgWgAwIBAgIUPxywpg3/+VHmj8jJSvK62XC06zMwDQYJKoZIhvcNAQEL"
+    "BQAwHjEcMBoGA1UEAwwTczJuVGVzdEludGVybWVkaWF0ZTAgFw0yMDAxMjQwMTEw"
+    "MjFaGA8yMTE5MTIzMTAxMTAyMVowGDEWMBQGA1UEAwwNczJuVGVzdFNlcnZlcjCC"
+    "ASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAJUbMpdROM6cjb8xgr5kgKHn"
+    "JVDfhbLg4pxBwWwlayb6/N60JLG9KzWAWhZBmz+Px6kr/1dL6+bL3mLuNBCQpYBS"
+    "Pee2n7KL9PvsMYZmnYFyn94bXbjBCRxGR+a9lcGHLlZ4C+rrLNi9pUwxf7VIRglR"
+    "zwHWAFg5xTX6lCmziNM4OMkq8lHkLopHDUg5yI4VTc3EEGqDIf3+0BheIHcUFbIW"
+    "kFOjRDdL3lMGKEj0+LErzzbhJczBlRMqSMiuYeaWgORLpRNtMeNmbR8oLJFchpF0"
+    "A9fIO2/Yg+nclcDDhsUBkkfcIKRySGDumKLuYM+hOHp5vQo8tcvyQ6s3U5YULQUC"
+    "AwEAAaNXMFUwDAYDVR0TAQH/BAIwADAdBgNVHQ4EFgQUkVKVmfjICpx4fkvJO6YJ"
+    "mdoKz3owDgYDVR0PAQH/BAQDAgPoMBYGA1UdJQEB/wQMMAoGCCsGAQUFBwMBMA0G"
+    "CSqGSIb3DQEBCwUAA4IBAQBXoWDI1Gi9snC4K6S7E0AoLmGEPUWzc4fd4Cbj9PRp"
+    "mSKpsJOYjmneIV34WqnvUXrBkkzblEb9RdszN96WuRAaZJQegRtKOWN5Iggd4sHM"
+    "8XEx/LeJHc08uSb2d/TnhhOPALoJl/w6M5e6yOezCEJorsOXuVBcbuEKfne7oMA1"
+    "GziFnVPtwiwXxsX16KilsQRylnK0bV/x1BOgYByCDcXorMndsAYjn4yG1D4l8TbC"
+    "kCtK1bafEVoASpOFQ8tSeOXBL7Fvw9mFFzs3/ajBTz2nBLDsnP8XH5C/vy8wNGSd"
+    "Tdcs7DRLYhNJxYopcMgCwyyCAtEFcHkovCSrJ6HUl/ko"
+    "-----END CERTIFICATE-----"
```

Comment:
For posterity, this cert decodes to:

Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            3f:1c:b0:a6:0d:ff:f9:51:e6:8f:c8:c9:4a:f2:ba:d9:70:b4:eb:33
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=s2nTestIntermediate
        Validity
            Not Before: Jan 24 01:10:21 2020 GMT
            Not After : Dec 31 01:10:21 2119 GMT
        Subject: CN=s2nTestServer
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (2048 bit)
                Modulus:
                    00:95:1b:32:97:51:38:ce:9c:8d:bf:31:82:be:64:
                    80:a1:e7:25:50:df:85:b2:e0:e2:9c:41:c1:6c:25:
                    6b:26:fa:fc:de:b4:24:b1:bd:2b:35:80:5a:16:41:
                    9b:3f:8f:c7:a9:2b:ff:57:4b:eb:e6:cb:de:62:ee:
                    34:10:90:a5:80:52:3d:e7:b6:9f:b2:8b:f4:fb:ec:
                    31:86:66:9d:81:72:9f:de:1b:5d:b8:c1:09:1c:46:
                    47:e6:bd:95:c1:87:2e:56:78:0b:ea:eb:2c:d8:bd:
                    a5:4c:31:7f:b5:48:46:09:51:cf:01:d6:00:58:39:
                    c5:35:fa:94:29:b3:88:d3:38:38:c9:2a:f2:51:e4:
                    2e:8a:47:0d:48:39:c8:8e:15:4d:cd:c4:10:6a:83:
                    21:fd:fe:d0:18:5e:20:77:14:15:b2:16:90:53:a3:
                    44:37:4b:de:53:06:28:48:f4:f8:b1:2b:cf:36:e1:
                    25:cc:c1:95:13:2a:48:c8:ae:61:e6:96:80:e4:4b:
                    a5:13:6d:31:e3:66:6d:1f:28:2c:91:5c:86:91:74:
                    03:d7:c8:3b:6f:d8:83:e9:dc:95:c0:c3:86:c5:01:
                    92:47:dc:20:a4:72:48:60:ee:98:a2:ee:60:cf:a1:
                    38:7a:79:bd:0a:3c:b5:cb:f2:43:ab:37:53:96:14:
                    2d:05
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Basic Constraints: critical
                CA:FALSE
            X509v3 Subject Key Identifier: 
                91:52:95:99:F8:C8:0A:9C:78:7E:4B:C9:3B:A6:09:99:DA:0A:CF:7A
            X509v3 Key Usage: critical
                Digital Signature, Non Repudiation, Key Encipherment, Key Agreement
            X509v3 Extended Key Usage: critical
                TLS Web Server Authentication
    Signature Algorithm: sha256WithRSAEncryption
         57:a1:60:c8:d4:68:bd:b2:70:b8:2b:a4:bb:13:40:28:2e:61:
         84:3d:45:b3:73:87:dd:e0:26:e3:f4:f4:69:99:22:a9:b0:93:
         98:8e:69:de:21:5d:f8:5a:a9:ef:51:7a:c1:92:4c:db:94:46:
         fd:45:db:33:37:de:96:b9:10:1a:64:94:1e:81:1b:4a:39:63:
         79:22:08:1d:e2:c1:cc:f1:71:31:fc:b7:89:1d:cd:3c:b9:26:
         f6:77:f4:e7:86:13:8f:00:ba:09:97:fc:3a:33:97:ba:c8:e7:
         b3:08:42:68:ae:c3:97:b9:50:5c:6e:e1:0a:7e:77:bb:a0:c0:
         35:1b:38:85:9d:53:ed:c2:2c:17:c6:c5:f5:e8:a8:a5:b1:04:
         72:96:72:b4:6d:5f:f1:d4:13:a0:60:1c:82:0d:c5:e8:ac:c9:
         dd:b0:06:23:9f:8c:86:d4:3e:25:f1:36:c2:90:2b:4a:d5:b6:
         9f:11:5a:00:4a:93:85:43:cb:52:78:e5:c1:2f:b1:6f:c3:d9:
         85:17:3b:37:fd:a8:c1:4f:3d:a7:04:b0:ec:9c:ff:17:1f:90:
         bf:bf:2f:30:34:64:9d:4d:d7:2c:ec:34:4b:62:13:49:c5:8a:
         29:70:c8:02:c3:2c:82:02:d1:05:70:79:28:bc:24:ab:27:a1:
         d4:97:f9:28